### PR TITLE
Brendan lewis solution

### PR DESCRIPTION
**Pull request by lewis2ba**

### CODE REVIEW

**Comment by lewis2ba** [comment]
https://github.com/lewis2ba/project1

File: README.md [hunk 1]
```diff
@@ -16,7 +16,24 @@ the square brackets (for multiple-choice questions).
...
+     clear: both;
+     height: 0;
+     }
+.clearfix { display: inline-block; }
+/* start commented backslash hack \*/
+* html .clearfix { height: 1%; }
+.clearfix { display: block; }
+/* close commented backslash hack */
+
+Apply the clear fix to the parent container element.
```
Comment (RobertAKARobin) [inline README.md:35]: Nice! This is a pretty advanced clearfix.

File: README.md [hunk 2]
```diff
@@ -38,7 +55,7 @@ Select 1:
 Describe the rules of scope in JavaScript.
 
 Your Answer:
-```text
+```Variables created in functions with var are only useable in that function, they are local variables. Variables created outside of functions are global variables. Variables created without var are global variables.
```
Comment (RobertAKARobin) [inline README.md:58]: Nice!

File: README.md [hunk 3]
```diff
@@ -51,7 +68,15 @@ the variable pizza inside your method.
 
 Your Answer:
 ```js
-// write code here
+    var pizza = {
+
+          temperature: 70;
```
Comment (RobertAKARobin) [inline README.md:73]: This should be a comma, not a semicolon.

File: README.md [hunk 4]
```diff
@@ -64,7 +89,13 @@ argument. Finally, demonstrate calling `doSomething` with a function.**
...
 Your Answer:
 ```js
-// write code here
+      function doSomething(thingToDo){
+
+          thingToDo();
+      }
+      doSomething(function(){
+        console.log("like this?")
+      })
```
Comment (RobertAKARobin) [inline README.md:98]: Yes, but I'd recommend putting the console logging function on a different line:

```
function whatever(){
  console.log("like this?")
}
doSomething(whatever);
```

**Comment by RobertAKARobin** [comment]
Nicely done!

**Pull request closed without merging**